packages/synapse-interface/components/StateManagedBridge/BridgeTransactionButton.tsx

```diff
@@ -162,6 +178,13 @@ export const BridgeTransactionButton = ({
       onClick: () => switchChain({ chainId: fromChainId }),
       pendingLabel: 'Switching chains',
     }
+  } else if (hasQuoteOutputChanged && !hasUserConfirmedChange) {
+    buttonProperties = {
+      label: 'Confirm new price',
```

Comment:
`bridgeModule` could change as well. Example: User first gets RFQ quote, and then inventory level changes. The quote is now through `CCTP`.

"Confirm new quote" (instead of price) covers both cases. Does the plumbing in the hook cover this as well?

Reply:
Good catch - confirming that the hook covers the scenario that `bridgeModule` changes while bridge selections remain the same as previous quote. 

The `useConfirmNewPrice()` hook will check for changes greater than 1bps given current and previous quotes share the same origin input amount, origin chainId, origin token, destination chainId, and destination token. 

Updated button text in f682d7337